engine/src/main/java/org/terasology/config/facade/BindsConfiguration.java

```diff
@@ -10,24 +10,54 @@
 import java.util.List;
 
 /**
- * Facade for {@link Config#getBinds()}
+ * Facade for {@link Config#getBinds()}.
+ * The binds configuration holds the mapping from bind uris.
+ * The {@link SimpleUri} for a binding contains the module from the binding and the id from the binding annotation, 
+ * e.g. {@link RegisterBindButton.
  */
 public interface BindsConfiguration {
 
+    /**
+     * Returns true if the input is bound to a bind uri.
+     */
     boolean isBound(Input newInput);
 
+    /**
+     * Sets all bindings to the mappings from the given config.
```

Comment:
I'd write: 

` * Sets all bindings to the mappings contained in the given config.`